src/coffea/nanoevents/factory.py

```diff
@@ -68,7 +67,7 @@ def _key_formatter(prefix, form_key, form, attribute):
     return prefix + f"/{attribute}/{form_key}"
 
 
-class _map_schema_base(ImplementsFormTransformation):
+class _map_schema_base:  # ImplementsFormMapping, ImplementsFormMappingInfo
```

Comment:
Any reason not to inherit the protocol(s)?

Reply:
They weren't exported yet! I think a new release includes them. Let me check.